src/argus_htmx/destinations/urls.py

```diff
@@ -1,21 +1,11 @@
-from django.http import HttpResponse
-from django.template import Template, RequestContext
 from django.urls import path
 
-
-def placeholder(request):
-    template = Template(
-        """{% extends "htmx/base.html" %}
-        {% block main %}
-        <h1>DESTINATION PLACEHOLDER</h1>
-        {% endblock main %}
-        """
-    )
-    context = RequestContext(request)
-    return HttpResponse(template.render(context))
-
+from . import views
 
 app_name = "htmx"
 urlpatterns = [
-    path("", placeholder, name="destination-placeholder"),
+    path("", views.destinations, name="destinations"),
+    path("create_destination/", views.destinations_create, name="destinations-create"),
+    path("update_destination/<int:pk>/", views.destinations_update, name="destinations-update"),
+    path("delete_destination/<int:pk>/", views.destinations_delete, name="destinations-delete"),
```

Comment:
POST to /destination/: create

POST to /destination/<int:pk>/: update

POST to /destination/<int:pk>/: delete

This is the old, well tested way.

Reply:
Made it like this, so now /destination/ is used to get the site itself with GET and POST to create a destination